src/components/BusinessViewMap.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import BusinessViewMap from '@components/BusinessViewMap'
+import { render } from '@testing-library/react-native'
+import React from 'react'
+import { ExampleLocationType } from '@components/BusinessView'
+
+const ExampleBusinessLocation : ExampleLocationType = {
+  id: '1',
+  address: 'Unit B Strahan House, Avon St, Bath BA1 1UN',
+  latitude: 51.37758520597919,
+  longitude: -2.3572347659685513
+}
+
+it('matches snapshot', () => {
+  const { toJSON } = render(<BusinessViewMap businessLocation={ExampleBusinessLocation}/>)
+  expect(toJSON()).toMatchSnapshot()
+})
```

Comment:
I'll add a few, missed this.

Reply:
I want to discuss this view because I wanted to be able to use the MapView for this view too to access the pins, etc by passing in certain parameters.

Reply:
Im not too sure on how this component works or some of its functionality,  but other tests could include checking that certain styles are correct when certain events occur or that certain functions are called when certain buttons are pressed etc.